### DISCUSSION TITLE
Exception 패키지

### DISCUSSION THREAD

**Discussion by Dae-Hwa** [Sc0up/scoup-backend#111] [Refactor]
각 기능 패키지 내부에 exception 패키지를 추가하는건 어떨까요?

**Comment by janeljs** [comment]
안 그래도 말씀드리려고 했어요!ㅎㅎ controller, service, mapper랑 같은 층위에 exception 패키지 추가해도 될까요?

**Reply by Dae-Hwa** [reply]
맞아요 그런 의미에요 @min27604 어떠신가요 ㅎㅎ
